keys/src/address.rs

```diff
@@ -223,7 +267,6 @@ mod tests {
             address.to_string(),
             "16meyfSoQV6twkAAxPe51RtMVz7PGRmWna".to_string(),
         );
-
```

Comment:
Don't introduce more unrelated changes without a good reason. It's against the consistency here, see the test below.